templates/pre-push

```diff
@@ -1,6 +1,26 @@
 #!/usr/bin/env bash
 set -e
 
+# Delegating script for multiple git pre-push hooks
+# This file will call any other pre-push.* files *except* pre-push.sample
+#
+# Called by "git push" after it has checked the remote status,
+# but before anything has been
+# pushed.  If this script exits with a non-zero status nothing will be pushed.
+#
+# This hook is called with the following parameters:
+#
+# $1 -- Name of the remote to which the push is being done
+# $2 -- URL to which the push is being done
+#
+# If pushing without using a named remote those arguments will be equal.
```

Comment:
Equal to what?

Reply:
To each other? Will just be the URL twice? How should we phrase it?